.github/workflows/rust-next.yml

```diff
@@ -7,22 +7,22 @@ jobs:
     name: Test
     strategy:
       matrix:
-        build: [linux, windows, mac, beta, nightly, minimal, default]
+        build: [stable, linux, windows, mac, nightly, minimal, default]
```

Comment:
> Run beta, nightly on Windows, osx?

I don't see nightly change here. I am assuming that we didn't add it since we decided to only test the next versions as part of this file?

Reply:
When making the change, I balanced simplicity with the desire and decided to keep the multi-platform focus on beta and a single on nightly, so it ends up being  a tapered approach.